doc/en/components/charts/chart-features-tooltips.md

```diff
@@ -7,8 +7,113 @@ mentionedTypes: []
+1. `Category` tooltips will display a grouped tooltip for all data points in the category that the pointer is positioned over.
+1. `None` will prevent tooltips from displaying.
+
+```razor
+<CategoryChart Height="500px" Width="700px"
+    DataSource="@DataSource"
+    ToolTipType="ToolTipType.Category"/>
+```
+
+```html
+<igx-category-chart
+    [dataSource]="data"
+    toolTipType="Category">
+</igx-category-chart>
+```
+
+```tsx
+ <IgrCategoryChart dataSource={this.state.data}
+    toolTipType="Category" />
+```
+
+```html
+<igc-category-chart
+    id="chart"
+    tool-tip-type="Category">
+</igc-category-chart>
+```
+
+<!--Angular-->
+# $PlatformShort$ Tooltip Templates
+
+The $PlatformShort$ category chart component provides default tooltips for each type of series. The default tooltips displays all the information relevant to the particular series item for example series title, data values, axis values. They are styled to match the series' style. If default tooltips are not sufficient, tooltip templates can be configured to customize the tooltip content and look and feel.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-tooltip-template" 
+           alt="$PlatformShort$ Tooltip Templates Example" 
+           github-src="charts/category-chart/tooltip-template">
+</code-view>
+
+<div class="divider--half"></div>
+
+The tooltip content is customized by creating a template for the tooltip as demonstrated in the following code.
+
+```html
+<ng-template let-series="series" let-item="item" #valueTooltip>
+   <div>
+      <span> {{item.Country}} energy use: <br/></span>
+      <span> Coal :{{item.Coal | number}}<br/></span>
+      <span> Oil :{{item.Oil | number}}<br/></span>
+      <span> Gas :{{item.Gas | number}}<br/></span>
+      <span> Nuclear :{{item.Nuclear | number}}<br/></span>
+      <span> Hydro :{{item.Hydro | number}}<br/></span>
+    </div>
+</ng-template>
+
+<div class="chart">
+    <igx-category-chart height="100%" width="100%"
+        [dataSource]="data" chartTitle="Energy Production"
+        subtitle="by Country"
+        [tooltipTemplate]="valueTooltip"
+        [chartType]="chartType">
+    </igx-category-chart>
+</div>
+```
+
+```tsx
+ <IgrCategoryChart height="100%" width="100%"
+        dataSource={this.state.data}
+        chartTitle="Energy Production"
+        subtitle="by Country"
+        tooltipTemplate={this.valueTooltip}
+        chartType={this.state.chartType} />
+```
+```html
+<igc-category-chart
+    id="chart"
+    width="800px"
+    height="700px"
+    chart-title="Energy Production"
+    subtitle="by Country"
+    chart-type="Column">
+  </igc-category-chart>
+```
+
+```ts
+let chart = (document.getElementById("chart") as IgcCategoryChartComponent);
+chart.dataSource = categoryData;
+
+let template: TemplateFunction;
+chart.tooltipTemplate = template;
+```
+<!--end:Angular-->
```

Comment:
why there is Angular build flag here?

Reply:
we don't have specifications for a tooltip template sample, nor is there ones for react, wc, or blazor.

Reply:
yes, we do
- [WC](https://github.com/IgniteUI/igniteui-wc-examples/blob/vnext/samples/charts/data-chart/series-tooltips/src/index.ts#L84)
- [REACT](https://github.com/IgniteUI/igniteui-react-examples/blob/vnext/samples/charts/data-chart/chart-overview/src/DataChartOverview.tsx#L118)
- [BLAZOR event trigger](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/data-chart/type-scatter-polygon-series/App.razor#L15)
- [BLAZOR event handler](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/data-chart/type-scatter-polygon-series/wwwroot/DataChartTypeScatterPolygonStyles.js#L34)

Reply:
Cannot use CategoryChart though. SeriesAdded fires but e.Series is not accessible and is always null.

http://tfs.infragistics.local:8080/tfs/Engineering/NetAdvantage/_workitems/edit/272693

Since the Template section in the docs uses Category Chart for Angular we can merge this as-is or wait until the other platforms are fixed and samples are working properly.